packages/rocketchat-wechat/common.js

```diff
@@ -0,0 +1,71 @@
+/* global CustomOAuth */
+import _ from 'underscore';
+
+const config = {
+	serverURL: 'https://open.weixin.qq.com',
```

Comment:
Are those setting values valid for all users of WeChat?

Reply:
@Hudell yes, the ones I am putting in here is valid for all users. The variables are appid, appsecret, and the name fields.